src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -52,6 +52,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\MemoryManager.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\TlsOverPerCoreLockedStacksArrayPool.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\Utilities.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\Text\FormattingHelpers.CountDigits.cs" />
```

Comment:
nit: sort order

Reply:
What's wrong with the sort order? Don't we normally put files in a directory before folders in that directory?

Reply:
I didn't know that was what we were doing.

Looking at this file, we seem to be following alphabetical order (only):
```csproj
    <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\UnicodeCategory.cs" />
    <Compile Include="$(MSBuildThisFileDirectory)System\Guid.cs" />
```

Reply:
FWIW VS inserts items with [simple string sort](https://github.com/Microsoft/msbuild/blob/6851538897f5d7b08024a6d8435bc44be5869e53/src/Build/Construction/ProjectItemGroupElement.cs#L119). 

Of course it won't ever insert into an imported file like this.